focus-group.js

```diff
@@ -109,3 +107,36 @@ export function menuKeyUX(options) {
     }
   }
 }
+
+export function findGroupNodeByEventTarget(eventTarget) {
+  let supportedRoles = {
+    'menuitem': ["menu", "menubar"],
+    'option': ["listbox"],
+    'tab': ["tablist"]
+  }
+  let itemRole = eventTarget.role
+  let groupRoles = supportedRoles[itemRole]
+  if (!groupRoles) return null
+
+  let result
+  groupRoles.forEach((role) => {
+    let node = eventTarget.closest(`[role=${role}]`)
+    if (node) {
+      result = node
+    }
+  })
+  return result
+}
+
+export function isHorizontalOrientation(group) {
+  let ariaOrientation = group.getAttribute('aria-orientation')
+  if (ariaOrientation === "vertical") return false
+  if (ariaOrientation === "horizontal") return true
+
+  if (!ariaOrientation) {
```

Comment:
Why we need this `if`?

Reply:
according to [the doc](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-orientation) the `aria-orientation` has 3 possible values:

![image](https://github.com/ai/keyux/assets/22315929/ad26e18b-7235-40c5-a32b-32c874464e42)

Those `if` statement needs to handle the `undefined` or default value. If I understand it correctly in case of `menubar` or `tablist` it should be **horizontal** unless explicitly set